py-scoring/src/ai/h2o/sparkling/Initializer.py

```diff
@@ -173,10 +173,11 @@ def __add_url_to_classloader(gateway, url):
         logger = Initializer.__get_logger(jvm)
         while loader:
             try:
+                urlClassLoaderClass = jvm.py4j.reflection.ReflectionUtil.classForName("java.net.URLClassLoader")
```

Comment:
is it possible this will actually load the class using a different classloader resulting in a different instance of the class object?